sdk/communication/tests.yml

```diff
@@ -57,10 +64,4 @@ stages:
     EnvVars:
       SKIP_PHONENUMBER_INTEGRATION_TESTS: TRUE
       SKIP_LIVE_TEST: TRUE
-      AZURE_SUBSCRIPTION_ID: $(acs-subscription-id)
-      COMMUNICATION_LIVETEST_CONNECTION_STRING: $(communication-livetest-connection-string)
-      COMMUNICATION_PHONE_NUMBER: $(communication-livetest-phone-number)
```

Comment:
Why are we removing those?

Reply:
These environment variables will be covered in the `sub-config-communication-services-cloud-test-resources-java`